zipline/data/minute_bars.py

```diff
@@ -1111,11 +1164,15 @@ def load_raw_arrays(self, fields, start_dt, end_dt, sids):
                         excl_slice = np.s_[
                             excl_start - start_idx:excl_stop - start_idx + 1]
                         values = np.delete(values, excl_slice)
+
                 where = values != 0
                 # first slice down to len(where) because we might not have
                 # written data for all the minutes requested
-                out[:len(where), i][where] = values[where]
-            if field != 'volume':
-                out *= self._ohlc_inverse
+                if field != 'volume':
```

Comment:
Why the change on how the data is written to the index?
I lean towards the original, where the assignment is done in one place, to minimize drift between OHLC and Volume with regard to the write indices.